src/app/shared/components/number-stepper/component.ts

```diff
@@ -116,14 +117,16 @@ export class NumberStepperComponent implements AfterViewInit, OnDestroy, Control
   }
 
   set value(val: number | string) {
-    const parsed = this.type === 'decimal' ? this._decimalPipe.transform(val, '1.0-4') : val?.toString();
-    if (!Number.isNaN(parsed) && parsed !== null && parsed !== undefined) {
+    const parsed =
+      this.type === 'decimal' ? this._decimalPipe.transform(val, '1.0-4')?.replace(/,/g, '') : val?.toString();
```

Comment:
What was the case which required to add `replace(/,/g, '')`?

Reply:
once we enter upto four digits `ex: 1111`, the method return `1,111`, which not supported by number input and cannot be parsed too, that's why I'm removing the `,` so the value can be parsed.

Reply:
But this is only for `decimal` type and we're not using that in Resource quotas. Also, `1.0-4` should not return value with `,`.
In the screenshot below I entered 9 digits without any issue (on dev)

![screenshot-localhost_8000-2022 08 16-14_47_58](https://user-images.githubusercontent.com/13975988/184850478-fc52c26b-db98-4d3b-9d40-eb65cde17848.png)

This is what Angular docs say about the `digitsInfo` parameter: https://angular.io/api/common/DecimalPipe#digitsinfo